### ISSUE TITLE
Airgeddon interface selection missing interfaces

### ISSUE THREAD

**Issue by AegrumPraesul** [v1s1t0r1sh3r3/airgeddon#146]
#### What is your airgeddon version?

v8.01

#### What is your Linux O.S. and which version?

Ubuntu 18.04

#### Which is the chipset of your wireless card?

rtl8723be

#### Describe the issue and the steps to reproduce it

My Airgeddon only shows my ethernet connection in the interface selection, not my wifif card
I switched Ubuntu versions to a dualbooted version instead of an external one, but my wifi card still doesnt show
I tried `iwconfig` and got this output
`root@Aegrum:~/airgeddon# iwconfig`
`lo        no wireless extensions.`
` `
`eno1      no wireless extensions.`

`wlo1      IEEE 802.11  ESSID:"Red_And_Black"  `
 `         Mode:Managed  Frequency:2.437 GHz  Access Point: 70:4D:7B:4D:08:40`   
  `        Bit Rate=72.2 Mb/s   Tx-Power=20 dBm   `
   `       Retry short limit:7   RTS thr=2347 B   Fragment thr:off`
    `      Encryption key:off`
     `     Power Management:on`
      `    Link Quality=70/70  Signal level=-24 dBm  `
       `   Rx invalid nwid:0  Rx invalid crypt:0  Rx invalid frag:0`
        `  Tx excessive retries:0  Invalid misc:1   Missed beacon:0`

I did post an earlier issue, however that one got closed due to a misunderstanding, still unable to resolve the issue

**Comment by v1s1t0r1sh3r3** [comment]
Put here the output (or screenshot) of this command: `ip link` and then the output of this other command `ip link | grep -E "^[0-9]+" | cut -d ':' -f 2 | awk '{print $1}' | grep lo -v`

This is the way how airgeddon detects the interfaces because is a standard working in all Linux. I'm downloading now the Ubuntu 18.04 iso to check if something new is "breaking" this. But in theory everything should be normal.

**Comment by AegrumPraesul** [comment]
The ip link output :
```
aegrum@Aegrum:~$ ip link
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
2: eno1: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc fq_codel state DOWN mode DEFAULT group default qlen 1000
    link/ether 6c:c2:17:5f:5f:92 brd ff:ff:ff:ff:ff:ff
5: wlo1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP mode DORMANT group default qlen 1000
    link/ether 90:48:9a:08:c6:11 brd ff:ff:ff:ff:ff:ff
```
output of the second command

```
aegrum@Aegrum:~$ ip link | grep -E "^[0-9]+" | cut -d ':' -f 2 | awk '{print $1}' | grep lo -v
eno1

```

**Comment by v1s1t0r1sh3r3** [comment]
omg, I know what is happening... the last part of the command is `grep lo -v` which is to discard the loopback interface. In your case, your wireless card contains "lo" in the name... so is discarded. I'll put this on the work queue to see if we can solve it. Thanks for the report.

Regards.

**Comment by v1s1t0r1sh3r3** [comment]
Fixed. It will be released on the new v8.10 which is incoming very soon. It was easy to fix.

You can already use it if you want instead of waiting to the release. Just perform a git clone using the `-b dev` to download airgeddon from the development branch. Closing.
